app/assets/stylesheets/home_page.scss

```diff
@@ -0,0 +1,337 @@
+// Place all the styles related to the HomePage controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;500;600;700;800&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200;0,300;0,400;0,600;0,700;0,800;0,900;1,200;1,300;1,400;1,600;1,700;1,800;1,900&display=swap");
+$color-blue: #659dbd;
+$color-red: #dc143c;
+$color-orange: #e89853;
+$color-grey: #d3d3d3;
+$color-grey-dark: #a9a9a9;
+$color-white: #fff;
+$color-black: #000;
+
+* {
+    box-sizing: border-box;
+}
+
+body {
+    display: flex;
+    min-height: 100vh;
+    flex-direction: column;
+    margin: 0;
+    padding: 0;
+    color: white;
+    font-family: "Montserrat", sans-serif;
+}
+
+ul {
+    list-style: none;
+}
+
+a {
+    color: white;
+    text-decoration: none;
+}
+
+.hero-wrapper {
+    background: asset-url("bg.jpg");
+    background-size: cover;
+
+    header {
+        display: flex;
+        justify-content: space-around;
+        align-items: center;
+
+        .logo {
+            display: flex;
+            align-items: center;
+            padding: 2.5em 0;
+            font-size: 14px;
+
+            .logo-mark {
+                img {
+                    width: 40px;
+                    height: 40px;
+                }
+            }
+
+            .logo-text {
+                padding-left: 0.7em;
+                font-weight: 600;
+            }
+        }
+
+        nav ul {
+            display: flex;
+            margin: 0;
+            padding: 0;
+            padding: 2.5em 0em;
+            font-size: 14px;
+
+            li:not(:first-child):before {
+                content: "";
+                background: asset-url("menudot.png") no-repeat;
+                opacity: 1;
+                background-size: 4px 4px;
+                background-position: 50% 50%;
+                padding: 0 1em;
+            }
+        }
+    }
+}
+
+.hero-content {
+    width: 40%;
+    justify-content: center;
```

Comment:
does `justify-content` here have any effect since you don't use `display: flex`?